src/improvements/tasks/sep/000-opcm-upgrade-v200/VALIDATION.md

```diff
@@ -0,0 +1,1042 @@
+                       Verify that the old implementation is set in this slot using:
+                        - <i>cast call 0x05F9613aDB30026FFd634f38e5C4dFd30a197Fa1 "gameImpls(uint32)(address)" 0 --rpc-url sepolia</i>
+                        - <i>cast storage 0x05F9613aDB30026FFd634f38e5C4dFd30a197Fa1 0xffdfc1249c027f9191656349feb0761381bb32c9f557e01f419fd08754bf5a1b --rpc-url sepolia</i> 
+                       The Raw Slot can be derived from:
+                        - <i>cast index uint32 0 101</i>
+
+  ----- DecodedStateDiff[5] -----
+    Who:               <a href="https://github.com/ethereum-optimism/superchain-registry/blob/b40cf4289c58e28eb1c791f9ad5724380b7516a7/superchain/configs/sepolia/op.toml#L58">0x16Fc5058F25648194471939df75CF27A2fdC48BC</a>
+    Contract:          OptimismPortal2 - OP Sepolia Testnet
+    Chain ID:          11155420
+
+    Raw Slot:          0x360894a13ba1a3210667c828492db98dca3e2076cc3735a920a3ca505d382bbc
+    Raw Old Value:     0x00000000000000000000000035028bae87d71cbc192d545d38f960ba30b4b233
+    Raw New Value:     0x0000000000000000000000002d7e764a0d9919e16983a46595cfa81fc34fa7cd
+
+    Decoded Kind:      address
+    Decoded Old Value: 0x35028bAe87D71cbC192d545d38F960BA30B4B233
+    Decoded New Value: <a href="https://github.com/ethereum-optimism/superchain-registry/blob/b40cf4289c58e28eb1c791f9ad5724380b7516a7/validation/standard/standard-versions-sepolia.toml#L31">0x2D7e764a0D9919e16983a46595CfA81fc34fa7Cd</a>
+    Summary:           ERC-1967 implementation slot
+    Detail:            Standard slot for storing the implementation address in a proxy contract that follows the ERC-1967 standard.
+
+  ----- DecodedStateDiff[6] -----
+    Who:               <a href="https://github.com/ethereum-optimism/superchain-registry/blob/b40cf4289c58e28eb1c791f9ad5724380b7516a7/validation/standard/standard-versions-sepolia.toml#L40">0x1B25F566336F47BC5E0036D66E142237DcF4640b</a>
+    Contract:          OPContractsManager - Sepolia Testnet
+    Chain ID:          11155111
+
+    Raw Slot:          0x0000000000000000000000000000000000000000000000000000000000000016
+    Raw Old Value:     0x0000000000000000000000000000000000000000000000000000000000000001
+    Raw New Value:     0x0000000000000000000000000000000000000000000000000000000000000000
+    [WARN] Slot was not decoded
+
+    Summary:           <i>isRC</i> storage slot updated to 0.
+    Detail:            Once OPContractsManager is upgraded, the <i>isRC</i> flag is set to false. 
+                       This happens in the first invocation of the <i>upgrade</i> function.
+                       Slot 22 is the <i>isRC</i> flag: <i>cast to-hex 22</i> -> <i>0x16</i>. 
+                       Please refer to <i>'Figure 0.1'</i> at the end of this report for the storage layout of OPContractsManager.
```

Comment:
as expected, this diff is not showing in tenderly. Should make it prominent that it may not appear.

Reply:
What do you think? https://github.com/ethereum-optimism/superchain-ops/pull/709/commits/c97270bf8a261a83f1ac5261fed81f090650aef6